README.md

```diff
@@ -5,6 +5,8 @@
 
 ## Recent updates
 
+* 2021-03-31: Release [`0.2.0` was published](https://github.com/bazelbuild/rules_python/releases/tag/0.2.0).
```

Comment:
Open to suggestions on what to include as detail here.

Reply:
IMO we should just remove the "Recent updates" section as it's totally redundant with the releases page. We could migrate the other content here to the Release it belongs on, or maybe leave here with "Releases prior to 2021 have their notes below"

Reply:
Agreed. I'll remove it.